composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Better Reflection - an improved code reflection API",
     "license": "MIT",
     "require": {
-        "php": "~8.0.12 || ~8.1.0 || ~8.2.0",
```

Comment:
Ah, just realized that we didn't update `composer.lock` :| 

Running Renovate with that now.